prody/proteins/interactions.py

```diff
@@ -0,0 +1,2034 @@
+                from simtk.openmm.app import PDBFile
+            
+            fixer = PDBFixer(filename=pdb)
+            fixer.findMissingResidues()
+            fixer.removeHeterogens(True)
+            fixer.findMissingAtoms()
+            fixer.addMissingAtoms()
+            fixer.addMissingHydrogens(pH)
+            PDBFile.writeFile(fixer.topology, fixer.positions, open('addH_'+pdb, 'w'))
+            LOGGER.info("Hydrogens were added to the structure. Structure {0} is saved in the local directry.".format('addH_'+pdb))
+
+        except ImportError:
+            raise ImportError('Install PDBFixer and OpenMM in order to fix the protein structure.')
+
+    else:
+        raise TypeError('Method should be openbabel or pdbfixer')
+    
+    
+def calcHydrogenBonds(atoms, distA=3.0, angle=40, cutoff_dist=20, **kwargs):
+    """Compute hydrogen bonds for proteins and other molecules.
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    
+    :arg distA: non-zero value, maximal distance between donor and acceptor.
+    :type distA: int or float, default is 3.0
+    
+    :arg angle: non-zero value, maximal (180 - D-H-A angle) (donor, hydrogen, acceptor).
+    :type angle: int or float, default is 40.
+    
+    :arg cutoff_dist: non-zero value, interactions will be found between atoms with index differences
+        that are higher than cutoff_dist.
+        default is 20 atoms.
+    :type cutoff_dist: int
+
+    Structure should contain hydrogens.
+    If not they can be added using addHydrogens(pdb_name) function available in ProDy after Openbabel installation.
+    `conda install -c conda-forge openbabel`
+    
+    Note that the angle which it is considering is 180-defined angle D-H-A (in a good agreement with VMD)
+    Results can be displayed in VMD by using showVMDinteraction() """
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+    
+    donors = kwargs.get('donors', ['N', 'O', 'S', 'F'])
+    acceptors = kwargs.get('acceptors', ['N', 'O', 'S', 'F'])
+    
+    if atoms.hydrogen == None or atoms.hydrogen.numAtoms() < 10:
+        LOGGER.info("Provide structure with hydrogens or install Openbabel to add missing hydrogens using addHydrogens(pdb_name) first.")
+    
+    contacts = findNeighbors(atoms, distA)
+    short_contacts = cleanNumbers(contacts)
+    pairList = [] # list with Donor-Hydrogen-Acceptor(indices)-distance-Angle
+    
+    LOGGER.info('Calculating hydrogen bonds.')
+    for nr_i,i in enumerate(short_contacts):
+        # Removing those close contacts which are between neighbour atoms
+        if i[1] - cutoff_dist < i[0] < i[1] + cutoff_dist:
+            pass
```

Comment:
you can use `continue` to continue on to the next round of the loop and then you don't need an `else` in the next line

Reply:
I've done this now